javaagent/build.gradle

```diff
@@ -62,11 +63,15 @@ shadowJar {
 
 //Includes instrumentations, but not exporters
 task lightShadow(type: ShadowJar) {
+  archiveClassifier.set("")
   dependsOn ':instrumentation:shadowJar'
   def projectsWithShadowJar = [project(':instrumentation')]
   with isolateSpec(projectsWithShadowJar)
 }
 
+// lightShadow is the default classifier we publish so disable the default jar.
+jar.enabled = false
```

Comment:
I have a feeling there was some luck involved in not overwriting lightShadow with this default jar and publishing it. By removing the `noShadowPublish` branch it became disallowed to have two tasks publishing the same file.